PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchSavedItem+ItemsListItem.swift

```diff
@@ -9,6 +9,16 @@ import Localization
 import SharedPocketKit
 
 extension SearchSavedItem: ItemsListItem {
+    var isCollection: Bool {
+        // TODO: Refactor when working on opening a collection tickets
+        guard let givenURL = item.asItem?.givenUrl, let url = URL(string: givenURL), url.host == "getpocket.com",
```

Comment:
To sidestep the duplicate code can you just use `return item.isCollection`?

Reply:
the `item` here does not have `isCollection` as its based on the GraphQL type and does not have that property. Definitely agree it would be better if this was refactored, but out of scope for native collections at this point.